src/rgw/rgw_rest.cc

```diff
@@ -784,7 +784,7 @@ int recv_body(struct req_state* const s,
   }
 }
 
-int RGWGetObj_ObjStore::get_params()
+int RGWGetObj_ObjStore::get_params(const Span& parent_span)
```

Comment:
why do you pass span to  functions that don't use it?

Reply:
My current implementation is invloves passing Span as optional_parameter to functions instead of the previous one which used stack. Majarity functions are just inherited from other classes so to make them compile I need to overload those functions too with spans and it includes functions which doesnt need tracing